packages/jest/src/types.ts

```diff
@@ -0,0 +1,3 @@
+// The type is used to replace the `any` type to be more explicit with return types.
+// We can't use the `Config` type from `jest` since this might be a breaking change.
```

Comment:
Why would it be a breaking change?

Reply:
It doesn’t ensure to be a breaking change. I’m just _afraid_ it to be a breaking change.

Reply:
Yeah I understand, I'm just not sure why 😁. Do you have a specific scenario in mind?

Reply:
I just thought initially that someone may `require` this package and might have issues with typings.

Reply:
But I ain't sure if someone will ever `require` it.